usage-viewer/README.md

```diff
@@ -0,0 +1,62 @@
+# Retool Usage Viewer - Identify top apps by pageviews and queries
+
+## Why you need this
+As an administrator of a self-hosted Retool instance, you want to understand the most frequently used apps in your instance. The Retool audit log contains the information you need, but it's difficult to access and make sense of.
+
+The Retool Usage Viewer makes your job easier by displaying app usage in a way that's easy to consume.
+
+## What does the Usage Viewer do?
+Usage Viewer is a ready-made app that lets you do these things:
+
+1. View all applications in your Retool instance, sorted by pageviews or queries
+2. Filter by app name, folder, and time range
+3. Identify top apps visually using built-in charts
+
+## Screenshots - TODO
+
+## Technical details and setup
+The Usage Viewer connects to the Postgres instance containing your [Retool storage database](https://docs.retool.com/docs/configuring-retools-storage-database), where a copy of all audit logs are stored. This database is named `hammerhead_production` and is only accessible when self-hosting Retool.
```

Comment:
They can change this when self hosting:

`hammerhead_production`, or the database name you've changed it to,

Reply:
Ah haaaa, I did not know that!